File: source/hildie/go/lib/lib_test.go
package lib

import (
	"testing"
)

func TestGreet(t *testing.T) {
	expected := "Hello from Hildie Go Library, World!"
	result := Greet("World")
	if result != expected {
		t.Errorf("Expected %s, got %s", expected, result)
	}
}

func TestAdd(t *testing.T) {
	result := Add(2, 3)
	if result != 5 {
		t.Errorf("Expected 5, got %d", result)
	}
}

File: source/hildie/node/src/lib.test.ts
import { test } from "node:test";
import assert from "node:assert";
import { greet, add } from "./lib.js";

test("greet", () => {
  const result = greet("World");
  const expected = "Hello from Hildie JavaScript Library, World!";
  assert.strictEqual(result, expected, `Expected "${expected}" but got "${result}"`);
});

test("add", () => {
  const result = add(2, 3);
  const expected = 5;
  assert.strictEqual(result, expected, `Expected ${expected} but got ${result}`);
});
